go.mod

```diff
@@ -1,3 +1,6 @@
 module github.com/MediaMath/goavro.v2
 
-require github.com/golang/snappy v0.0.0-20170215233205-553a64147049
+require (
+	github.com/golang/snappy v0.0.0-20170215233205-553a64147049
+	gopkg.in/linkedin/goavro.v1 v1.0.5
```

Comment:
What on earth is this doing here?

Reply:
https://github.com/MediaMath/goavro.v2/blob/master/helperV1_test.go#L17

that whole file gets removed in later versions